api/opencl_runtime_layer.txt

```diff
@@ -1954,11 +1954,11 @@ descriptor information associated with mem_object.
 Image elements are stored according to their image format as described in
 <<image-format-descriptor, Image Format Descriptor>>.
 
-If the buffer object specified by mem_object is created with
+If the buffer object specified by _mem_object_ was created with
 CL_MEM_USE_HOST_PTR, the _host_ptr_ specified to *clCreateBuffer* must be
-aligned to the minimum of the *CL_DEVICE_IMAGE_BASE_ADDRESS_ALIGNMENT* value
+aligned to the maximum of the *CL_DEVICE_IMAGE_BASE_ADDRESS_ALIGNMENT* value
```

Comment:
Note, this does chance `minimum` to `maximum`.  I think that `minimum` is a typo.  The extension spec already says `maximum`.

Reply:
Maximum definitely makes more sense to me.